source/solutions-library/media-personalization.txt

```diff
@@ -36,172 +36,166 @@ Media
+
+- **Integration with multiple sources:** The system uses Atlas
+  Vector Search to pull content from a variety of articles and data
+  sources. This content is then compiled into dossiers, which provide
+  users with a detailed and contextual exploration of topics and are
+  curated to offer a narrative or analytical perspective beyond the
+  original content.
+
+- **Customizable output:** You can customize the system output by setting
+  parameters based on your audience's preferences or your specific project
+  requirements. This includes adjusting the level of detail, the amount of
+  technical terms, and the inclusion of multimedia elements.
+
+Build the Solution
+------------------
+
+You can view a solution demo at `<https://ist.media>`__.
+
+.. figure:: /includes/images/industry-solutions/ist-media-interface.svg
    :figwidth: 750px
    :alt: An image showing the homepage of the ist website
 
    Figure 2. Media platform homepage interface
 
-The underlying data model is straightforward; a representative news article looks
-like this:
+In the underlying data model, a representative news article has the following
+structure:
 
-.. figure:: /includes/images/industry-solutions/demo-model.svg
+.. figure:: /includes/images/industry-solutions/document-model-media.svg
    :figwidth: 750px
    :alt: An image showing an example of a news article in data model format
 
    Figure 3. Data model for a news article
 
 You can use `Voyage AI <https://www.voyageai.com/>`__ to generate your
-embeddings. To perform vector search, we created a vector index in the
-MongoDB Atlas web interface with the following configuration:
+embeddings. To perform vector search, create a vector index in MongoDB Atlas
+with the following configuration:
 
 .. figure:: /includes/images/industry-solutions/vector-index.svg
    :figwidth: 750px
    :alt: An image showing an embedding model
 
    Figure 4. Vector index for the embedding model
 
-Technologies and products used
-------------------------------
+Key Learnings
+-------------
+
+Below are core takeaways from this solution:
 
-MongoDB Developer Data Platform
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- You can **build an AI-powered media solution with MongoDB Atlas** to 
+  deliver tailored content to your users and automate backend processes,
+  such as keywords automation.
 
-- `Atlas Database <https://www.mongodb.com/products/platform/atlas-database>`__
-- `Atlas Vector Search <https://www.mongodb.com/products/platform/atlas-vector-search>`__
-- `MongoDB PyMongo Driver <https://www.mongodb.com/docs/languages/python/pymongo-driver/current/>`__
+- You can **store diverse media data with MongoDB's flexible document
+  model**, including user data, news articles and embeddings to develop
+  AI-driven applications.  
+
+- You can **personalize user content experiences with MongoDB Atlas
+  Vector Search**, aligning individual preferences and past interactions
+  to provide better content journeys.
```

Comment:
Do we want to rewrite these to fit the format of other key learnings sections? e.g. **sentence:** explanation